### ISSUE TITLE
Make settings names more unusual

### ISSUE THREAD

**Issue by earldouglas** [earldouglas/xsbt-web-plugin#250]
#189 sprung up again, this time with the `options` setting also used by sbt-progarded

**Comment by earldouglas** [comment]
This is available now in the [first milestone release for 1.2.0](https://bintray.com/earldouglas/sbt-plugins/xsbt-web-plugin/1.2.0-M1/view):

*project/plugins.sbt:*

```scala
addSbtPlugin("com.earldouglas" % "xsbt-web-plugin" % "1.2.0-M1")
```

**Comment by earldouglas** [comment]
This is now available in release version 1.1.1:

project/plugins.sbt:

addSbtPlugin("com.earldouglas" % "xsbt-web-plugin" % "1.1.1")
